website/content/docs/use-cases/hcp-hashistack-quickstart.mdx

```diff
@@ -0,0 +1,495 @@
+}
+
+resource "aws_vpc_peering_connection_accepter" "peer" {
+  vpc_peering_connection_id = hcp_aws_network_peering.peer.provider_peering_id
+  auto_accept               = true
+  accepter {
+    allow_remote_vpc_dns_resolution = true
+  }
+}
+
+resource "aws_route" "hvn-peering" {
+  route_table_id            = module.vpc.public_route_table_ids[0]
+  destination_cidr_block    = hcp_hvn.xx_2048_hvn.cidr_block
+  vpc_peering_connection_id = aws_vpc_peering_connection_accepter.peer.id
+```
+
+## Define HCP Consul
+
+HCP Consul enables you to quickly deploy Consul servers in AWS while offloading the operations burden to the SRE experts at HashiCorp.
+
+Provisioning HCP Consul is incredibly easy.
+You can define the cluster in Terraform using the [`hcp_consul_cluster`](https://registry.terraform.io/providers/hashicorp/hcp/latest/docs/resources/consul_cluster) resource,
+as shown in `hcp.tf`:
+
+```terraform
+resource "hcp_consul_cluster" "xx_2048_consul" {
+  hvn_id          = hcp_hvn.xx_2048_hvn.hvn_id
+  cluster_id      = "xx-2048-consul"
+  tier            = "development"
+  public_endpoint = true
+}
+```
+
+You can also provision a Consul cluster with one click in the HCP UI.
+
+## Define HCP Vault
+
+Defining a Vault cluster in Terraform is as simple as with Consul,
+using the [`hcp_vault_cluster`](https://registry.terraform.io/providers/hashicorp/hcp/latest/docs/resources/vault_cluster) resource.
+
+```terraform
+resource "hcp_vault_cluster" "xx_2048_vault" {
+  hvn_id          = hcp_hvn.xx_2048_hvn.hvn_id
+  cluster_id      = "xx-2048-vault"
+  public_endpoint = true
+}
+```
+
+### Vault Cluster Configurations
+
+We will further configure Vault to communicate with Nomad.
+The [Vault provider](https://registry.terraform.io/providers/hashicorp/vault/latest/docs) allows Terraform to read from, write to, and configure HashiCorp Vault:
+
+```terraform
+provider "vault" {
+  address   = hcp_vault_cluster.xx_2048_vault.vault_public_endpoint_url
+  namespace = "admin"
+  token     = hcp_vault_cluster_admin_token.xx_2048_vault_token.token
+}
+```
+
+You can find the detailed Vault configurations in [`vault-config.tf`](https://github.com/xiaolin-ninja/2048_hcp_hashistack/blob/main/vault-config.tf)
+
+## Define Nomad Configurations
+
+In [`nomad.tf`](https://github.com/xiaolin-ninja/2048_hcp_hashistack/blob/main/nomad.tf),
+we configure the AWS Security Group, find a demo AMI, and configure the Nomad server, client, and auto-scaling.
+The scripts `user-data-client.sh` and `user-data-server.sh` are run directly in the EC2 instances spun up by Terraform to provision Nomad.
+
+~> **Note:** Since this is a use-case demo, I am not following Security best-practices.
+For example, my AWS Security Group allows all ingress and egress traffic.
+I am also provisioning only one Nomad server and client cluster each, which does not insulate against downtime.
+For real production use cases, please properly configure your security group and have multiple clusters.
```

Comment:
I'd recommend re-wording this to remove `I` and be more generic. There might be other places where this needs to happen too. These use-cases won't have authors attached on our website, so the voicing doesn't make sense.